### PR TITLE
Revisions based on reviewer comments

### LINKED ISSUE #3: Address reviewers comments
I have received comments from the reviewers, and revisions are due to the editor by 10/3 (can get an extension if needed). 
[ReviewerComments.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7122308/ReviewerComments.pdf)

### PR DESCRIPTION
**Pull request by abachma2**
I have made revisions based on the reviewer comments, given in Issue #3. 
PDF of paper:
[revisions.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7188345/revisions.pdf)
PDF of response to reviewers:
[revise.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7188346/revise.pdf)
I tried to make a diff.tex file (hence the duplicate files in the `revise/` directory, but my LaTeX on my personal laptop does not want to build that right now. I will upload that on Monday when I get back to my desktop.

### CODE REVIEW

**Comment by abachma2** [comment]
I was able to get the diff file to compile with some help from @munkm. I also made some changes to my language to remove some of the passive voice and extra words.
Updated pdf:
[revisions.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7196466/revisions.pdf)
Updated diff.pdf:
[diff.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7196468/diff.pdf)

**Comment by nsryan2** [comment]
[Bachmann_epjn_revisions 2.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7198370/Bachmann_epjn_revisions.2.pdf)
I missed your update with the diff version, but here are my comments

**Comment by abachma2** [comment]
I have updated the paper based on comments from @nsryan2. 
Updated paper pdf:
[revisions.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7198824/revisions.pdf)

**Comment by robfairh** [comment]
Here is my review of this latest version:

> I have updated the paper based on comments from @nsryan2.
> Updated paper pdf:
> [revisions.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7198824/revisions.pdf)

* No need to include my middle name (Enrique).

* Maybe mention in the abstract that you are using Cyclus for the simulations.

* Table 2 extends beyond the margins.

* Line 100: grwoth

* Figure 1: TRSIO

* Scenario 2: There used to be 109 LWRs. 9,182 MMRs are deployed in the transition. The latter is 2 orders of magnitudes greater (You mention that it is one order only). This makes sense, cause the ratio of power is 1000/10 = 100.

* Line 190: extraneous article: 'an the advanced reactors'. Did you mean 'in'?

* Results:
Your description of the results is very thorough.
However, the wording is a bit verbose and repetitive.
Reducing the length of some sentences will enhance the readability of the paper.

As an example:

"The SWU capacity required to enrich uranium for the MMRs is greater than the capacity needed to enrich uranium for the LWRs."

The MMR fleet requires a greater SWU capacity than the LWR fleet.


* Figure 7: The subfigures are not vertically aligned.

* Figure 19: Overlapping of the figure titles.
Also, the ylabel in (a) and (b) differ.

* Conclusions:

"We used the current fleet of LWRs, without a transition to an advanced reactor for comparison."
Maybe: "We used the current fleet of LWRs, without a transition to an advanced reactor fleet for comparison."
Otherwise it sounds that the transition would deploy only one unit.

I like the conclusions. They nicely summarize the results.

* Future work: 

This is splendid. Good job.

**Review by munkm — APPROVED** [review]

**Comment by ZoeRichter** [comment]
[revisions-3-reviewed.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7263617/revisions-3-reviewed.pdf)

I added my comments as, well, comments!  I like the paper - sorry I took so long!

**Review by ZoeRichter — APPROVED** [review]
My feedback is in the pdf I included in my other comment.  I think once the last set of revisions go through, it'll be good to go!

**Comment by abachma2** [comment]
I have made revisions based on @ZoeRichter's comments. I tried to make the figures have a better resolution, but I don't see much of a difference unless I change the file type to PDF. Should I do that for all of the figures?

Updated pdf, with Figure 2 as a pdf:
[revisions.pdf](https://github.com/arfc/2021-bachmann-epjn/files/7279256/revisions.pdf)

**Comment by abachma2** [comment]
I plan to submit these revisions tomorrow (10/5) unless you have any more comments @robfairh @munkm @ZoeRichter

**Comment by ZoeRichter** [comment]
I think Figure 2 looks great that way, if you can do the others it should help!

**Comment by munkm** [comment]
I have a few suggestions for your responses to the reviewers: 

In general, try to be specific with the sub- and subsub-section numbers where you've made the additions they've requested. If you've added figures or tables, include their numbers as well. That way your reviewers don't have to search to find the information they've requested. 

R1Q1: "A short discussion of this has been added to the results section of the paper, Section 3.6. The discussion includes teh design and enriched uranium stockpile of a theoretical enrichment facility to meet the needs of Scenario 2, but leaves optimization of meeting the demand to future work." -- remove "short" here; it doesn't enhance your message. 

R1Q2:

R2Q1: "An additional subsection of results have also been added to compare all of the scenarios together. This includes additional  gures that show the uranium mass and SWU capacity required for all reactors and just the advanced reactors in each scenario and a  gure to compare the total uranium mass in each scenarios." -- include subsection numbers and figure numbers where these are added

R2Q2: "Therefore, additional information about the reactors was not added to the paper. " -- I think you can say something like, "We chose not to add additional information about the reactors to the papers as all information required to replicate the work is provided. "

R2Q3:
"A discussion of how the refueling masses were calculated for use in the Cyclus simulations was added." -- say exactly where
 "These calculations were based on previous work done and these discussions were written by other people in the research group, leading to the addition of new authors on the paper." -- I think you could rephrase this to "These calculations were based on previous work performed by other members in the research group. We have added discussions about this work and have added them as additional authors on the paper."

**Comment by abachma2** [comment]
I have submitted these revisions to the journal. thank you so much for all of the help everyone!
